_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+
+The following categories are in-scope for the pyOpenSci review process:
+* **Data retrieval:** refers to packages for accessing and downloading data from online sources, including wrappers for accessing APIs.
+* **Data extraction:** these packages aid in retrieving data from unstructured sources such as text, images, and PDFs. They might also parse scientific data types and outputs from scientific equipment.
+* **Data processing/munging:** this category focuses on tools for handling data in specific formats that scientists may be interested in working with.
+* **Data deposition:** includes tools for depositing data into scientific research repositories.
+* **Data validation and testing:** refers to tools that enable automated validation and checking of data quality and completeness that in turn support scientific workflows.
+* **Data visualization:** these are packages that enhance a scientist’s experience in visualizing and analyzing data. If your package falls into the category of data visualization, please fill out a [pre-submission inquiry](https://github.com/pyOpenSci/software-submission/issues/new?assignees=&labels=0%2Fpresubmission&template=presubmission-inquiry.md&title=) first.
+* **Workflow automation:** these tools automate and link together workflows, and as such support reproducible workflows. These tools may include build systems and tools to manage continuous integration, as well as support version control.
+* **Citation management and bibliometrics:** these are packages that facilitate managing references.
+* **Scientific software wrappers:** these tools provide a Python interface for existing scientific packages written in other languages.
+* **Database interoperability:** these packages deal with bindings and wrappers for database APIs.
+
+Similar to the previous section, you can indicate which categories are applicable to your package in one of two ways:
+* adding an "X" (without quotation marks) inside the brackets in the Markdown version of the issue, so that it reads like this: [X]
+* submitting the issue without making any changes to this section, and then clicking on the checkboxes when they're rendered in the submitted issue.
+
+You can read more about each of these categories in the [Package Scope](https://www.pyopensci.org/software-peer-review/index.html) section of the [pyOpenSci Peer Review Guide](https://www.pyopensci.org/software-peer-review/about/package-scope.html).
+
+### Domain specific
+
+This section is currently evolving, and it's OK if your package doesn't fit into a specific listed domain.
+
+### Community Partnerships
+
+Similar to the Domain specific section, our Community Partnerships are continuously growing and changing, and it's OK if your package isn't affiliated with one of the listed partners.
```

Comment:
This is an ideal spot to write 2-4 sentences on the peer review partnership program why it's important and what value it adds.  link to the program and add a graphic. and then tell them if they are related to any of the current partners they should check the box. if they aren't sure they can ping an editor.